src/sidebar/components/annotation-omega.js

```diff
@@ -25,6 +26,18 @@ function AnnotationOmega({
   // TODO: `isEditing` will also take into account `isSaving`
   const isEditing = !!draft;
 
+  const annotationState = () =>
```

Comment:
I may opt to create a store selector soon that returns draft-or-state for a given annotation; would consider this prototyped code for the moment.